TESTS/mbed_functional/callback_big/main.cpp

```diff
@@ -23,180 +23,354 @@ using namespace utest::v1;
+T static_func2(T a0, T a1)
+{
+    return 0 | a0 | a1;
+}
 template <typename T>
-T static_func3(T a0, T a1, T a2) { return 0 | a0 | a1 | a2; }
+T static_func3(T a0, T a1, T a2)
+{
+    return 0 | a0 | a1 | a2;
+}
 template <typename T>
-T static_func4(T a0, T a1, T a2, T a3) { return 0 | a0 | a1 | a2 | a3; }
+T static_func4(T a0, T a1, T a2, T a3)
+{
+    return 0 | a0 | a1 | a2 | a3;
+}
 template <typename T>
-T static_func5(T a0, T a1, T a2, T a3, T a4) { return 0 | a0 | a1 | a2 | a3 | a4; }
+T static_func5(T a0, T a1, T a2, T a3, T a4)
+{
+    return 0 | a0 | a1 | a2 | a3 | a4;
+}
 
 // class functions
 template <typename T>
 struct Thing {
     T t;
     Thing() : t(0x80) {}
 
-    T member_func0() { return t; }
-    T member_func1(T a0) { return t | a0; }
-    T member_func2(T a0, T a1) { return t | a0 | a1; }
-    T member_func3(T a0, T a1, T a2) { return t | a0 | a1 | a2; }
-    T member_func4(T a0, T a1, T a2, T a3) { return t | a0 | a1 | a2 | a3; }
-    T member_func5(T a0, T a1, T a2, T a3, T a4) { return t | a0 | a1 | a2 | a3 | a4; }
-
-    T const_member_func0() const { return t; }
-    T const_member_func1(T a0) const { return t | a0; }
-    T const_member_func2(T a0, T a1) const { return t | a0 | a1; }
-    T const_member_func3(T a0, T a1, T a2) const { return t | a0 | a1 | a2; }
-    T const_member_func4(T a0, T a1, T a2, T a3) const { return t | a0 | a1 | a2 | a3; }
-    T const_member_func5(T a0, T a1, T a2, T a3, T a4) const { return t | a0 | a1 | a2 | a3 | a4; }
-
-    T volatile_member_func0() volatile { return t; }
-    T volatile_member_func1(T a0) volatile { return t | a0; }
-    T volatile_member_func2(T a0, T a1) volatile { return t | a0 | a1; }
-    T volatile_member_func3(T a0, T a1, T a2) volatile { return t | a0 | a1 | a2; }
-    T volatile_member_func4(T a0, T a1, T a2, T a3) volatile { return t | a0 | a1 | a2 | a3; }
-    T volatile_member_func5(T a0, T a1, T a2, T a3, T a4) volatile { return t | a0 | a1 | a2 | a3 | a4; }
-
-    T const_volatile_member_func0() const volatile { return t; }
-    T const_volatile_member_func1(T a0) const volatile { return t | a0; }
-    T const_volatile_member_func2(T a0, T a1) const volatile { return t | a0 | a1; }
-    T const_volatile_member_func3(T a0, T a1, T a2) const volatile { return t | a0 | a1 | a2; }
-    T const_volatile_member_func4(T a0, T a1, T a2, T a3) const volatile { return t | a0 | a1 | a2 | a3; }
-    T const_volatile_member_func5(T a0, T a1, T a2, T a3, T a4) const volatile { return t | a0 | a1 | a2 | a3 | a4; }
+    T member_func0()
+    {
+        return t;
+    }
+    T member_func1(T a0)
+    {
+        return t | a0;
+    }
+    T member_func2(T a0, T a1)
+    {
+        return t | a0 | a1;
+    }
+    T member_func3(T a0, T a1, T a2)
+    {
+        return t | a0 | a1 | a2;
+    }
+    T member_func4(T a0, T a1, T a2, T a3)
+    {
+        return t | a0 | a1 | a2 | a3;
+    }
+    T member_func5(T a0, T a1, T a2, T a3, T a4)
```

Comment:
These not look as neat as the old version, but if this is the coding style rules, I have no objection to it.